src/main/java/com/jersey/representations/Cook.java

```diff
@@ -15,11 +15,11 @@
     @JoinColumn(name = "person_id")
     private Long person_id;
 
-    @OneToMany(fetch = FetchType.LAZY)
```

Comment:
yes sure, lazy wasn't working and eager was the only one I found that worked. Didn't know there was any other option